.github/workflows/ci.yml

```diff
@@ -24,3 +24,31 @@ jobs:
         uses: actions/checkout@v2
       - name: Run tests
         run: sbt -v +test
+  deploy:
+    name: Package and Deploy
+    needs: test
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
```

Comment:
Why `@v3` here but `@v2` on line 19?

Reply:
No reason. Have now updated all actions to v3.

Reply:
Confirmed.